app/src/main/java/org/p2p/wallet/infrastructure/network/NetworkModule.kt

```diff
@@ -121,6 +122,15 @@ object NetworkModule : InjectionModule {
                 interceptor = null
             )
         }
+
+        single(named(SwapModule.JUPITER_RETROFIT_QUALIFIER)) {
```

Comment:
@NikolaiEmelianov there is a tradition to declare retrofit instances in NetworkModule only fyi